CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ and this project adheres to
 
 ### Added
 
+- [#4834](https://github.com/firecracker-microvm/firecracker/pull/4834): Add
+  RX_MRGBUF support to the `virtio-net` device. When this flag is enabled
+  guest network driver can perform better buffer management which in turn
+  improves RX and TX performance.
```

Comment:
```suggestion
- [#4834](https://github.com/firecracker-microvm/firecracker/pull/4834): Add
  support for `VIRTIO_NET_F_MRG_RXBUF` for the `virtio-net` device. When this
  feature is negotiated, guest `virtio-net` driver can perform better memory
  management which in turn improves RX and TX performance.
```



`RX_MRGBUF` is not a valid identifier. Either use, `VIRTIO_NET_F_MRG_RXBUF` or "mergeable buffers". I just saw you used it in the commit message bodies as well. Please fix it there, too.